docs/output.md

```diff
@@ -1,45 +1,301 @@
+
+#### Output files - genome-properties
+
+This subdirectory contains the results of running `Genome Properties` on the assembly’s proteins, detailing various protein signature sets present in the assembly. The results are provided in three formats.
+
+- **ERZ12345_gp.json.gz**: A JSON format file containing Genome Properties annotations
+- **ERZ12345_gp.tsv.gz**: A `tsv` file containing Genome Properties annotations in a tab-separated format.
+- **ERZ12345_gp.txt.gz**: A plain text file containing the Genome Properties annotations.
+
+#### Output files - kegg-modules
+
+This subdirectory contains results from evaluating which KEGG modules and pathways are represented in the assembly based on the presence of KEGG Orthologs (KOs). For each module, a completeness score is calculated depending on how many required KOs are detected.
+
+- **ERZ12345_kegg_modules_per_contigs.tsv.gz**: This `tsv` file contains the different KEGG modules and their completeness, including the contig they were found on.
+- **ERZ12345_kegg_modules_per_contigs.tsv.gz.gzi**: This file is an index for the per-contig KEGG modules file.
+- **ERZ12345_kegg_modules_summary.tsv.gz**: This `tsv` file contains a summary of the KEGG modules and their completeness for the whole assembly, not on a per-contig basis.
+- **ERZ12345_kegg_modules_summary.tsv.gz.gzi**: This file is an index for the summary KEGG modules file.
+
+#### Output files - dram-distill
+
+This subdirectory contains the outputs of running `DRAM-distill` on the KO hits per contig, the InterProscan and run_DBCan. DRAM generates summary reports and visualisations for the annotations on a per-assembly basis, into four different files.
+
+- **ERZ12345_dram.tsv.gz**: This `tsv` file contains the product of `DRAM-distill` for the assembly, including functions that were detected.
+- **ERZ12345_dram.html.gz**: This `html` file contains a heatmap visualisation of the detected functions by `DRAM-distill`.
+- **ERZ12345_genome_stats.tsv.gz**: This `tsv` file contains a summary of the assembly quality, including number of scaffolds.
+- **ERZ12345_metabolism_summary.xlsx.gz**: This `xlsx` file is an Excel spreadsheet containing information about annotations that represent common metabolisms.
+
+### gff
+
+The `gff` directory contains a single file that summarises most of the functional annotations generated by the pipeline on a per-protein basis into a single file.
+
+```bash
+├── qc
+├── cds
+├── taxonomy
+├── functional-annotation
+├── pathways-and-systems
+└── gff
+    └── ERZ12345_annotation_summary.gff.gz
+```
+
+#### Output files
+
+- **ERZ12345_annotation_summarygz**: This `gff` file contains an expansive and large summary of most of the functional annotations each protein has into a single GFF3 format file.
```

Comment:
We might want to have a table similar to this one: https://github.com/EBI-Metagenomics/mettannotator?tab=readme-ov-file#data-sources

Reply:
Yes, in another PR.

Reply:
https://embl.atlassian.net/browse/EMG-8058